packages/api-docs-builder/utils/replaceUrl.ts

```diff
@@ -63,8 +63,7 @@ export const replaceAPILinks = (url: string) => {
     .replace(
       /\/api\/(unstable-trap-focus|click-away-listener|no-ssr|portal|textarea-autosize)(.*)/,
       '/base/api/$1$2',
-    )
-    .replace(/\/api\/([^/]+)(-unstyled)(.*)/, '/base/api/$1$3');
+    );
```

Comment:
Let's please create an issue for this method, as it creates links in the previous format.